docs/_data/toc.yml

```diff
@@ -113,196 +119,199 @@
-        - title: Network Service Mesh
-          url: extensibility/adapters/nsm
-        - title: NGINX Service Mesh
-          url: extensibility/adapters/nginx-sm
-        - title: Nighthawk
-          url: extensibility/adapters/nighthawk
-        - title: Tanzu
-          url: extensibility/adapters/tanzu-sm
-        - title: Traefik Mesh
-          url: extensibility/adapters/traefik-mesh
-    - title: "Integrations"
-      url: extensibility/integrations
-    - title: "Load Generators"
-      url: extensibility/load-generators
-    - title: "Providers"
-      url: extensibility/providers
-    - title: "UI Plugins"
-      url: extensibility/ui
-    - title: "🦾 Extensions ↆ"
+    - title: Integrations
+      url: extensions/integrations
+    - title: Extensions ↆ
       url: extensions
       children:
-        - title: "Plugin: Kanvas"
+        - title: Helm Kanvas Snapshot
+          url: extensions/helm-kanvas-snapshot
+        - title: MeshSync Snapshot
+          url: extensions/kubectl-meshsync-snapshot
+        - title: Kanvas Design Tools ↆ
           url: extensions/kanvas
-          # grandchildren:
-          # - title: "Sharing a Design"
-          #   url: extensions/sharing-a-design
-        - title: "GitOps: Snapshots"
-          url: extensions/snapshot
+          grandchildren:
+            - title: Component Shape Guide
+              url: extensions/component-shape-guide
+            - title: Edges Style Guide
+              url: extensions/edges-shape-guide
+            - title: Import & Export Designs
+              url: extensions/import-export-designs
+            - title: Merging Designs
+              url: extensions/merging-design
+            - title: Publishing Designs
+              url: extensions/publishing-a-design
+            - title: Sharing Designs
+              url: extensions/sharing-a-design
+            - title: Working with Tags
+              url: extensions/working-with-tags
+        - title: Kanvas Snapshot
+          url: extensions/kanvas-snapshot 
 
-      # children:
-      # - title: Building a Remote Provider
-      #   url: extensibility/providers/building-a-remote-provider
 - title: 🧑‍💻 Contributing & Community
   url: project
   links:
-    - title: Community
+    - title: Meshery Community
       url: project/community
-    - title: Contributing ↆ
+    - title: Meshery Playground
+      url: installation/playground
+    - title: Contributing Guides ↆ
       url: project/contributing
       children:
-        - title: "Build & Release (CI) ↆ"
+        - title: Build & Release (CI) ↆ
           url: project/contributing/build-and-release
-          grandchildren:
-            - title: "End-to-End Testing"
-              url: project/contributing/contributing-ui-tests
-        - title: Adapters
+        - title: Contributing to Adapters
           url: project/contributing/contributing-adapters
-        - title: CLI ↆ
+        - title: CLI Contributions ↆ
```

Comment:
Stylistically, if our choice is to apply redundant wording to the section titles, why then, don't each of the Troubleshooting pages include "Troubleshooting" in there name (e.g. "Meshery Server Troubleshooting")?

I suggest that we leave "Contributions" out.